reporter/html/src/test/java/org/itsallcode/openfasttrace/report/html/view/html/MarkdownSpanConverterTest.java

```diff
@@ -0,0 +1,65 @@
+package org.itsallcode.openfasttrace.report.html.view.html;
+
+/*-
+ * #%L
+ * OpenFastTrace HTML Reporter
+ * %%
+ * Copyright (C) 2016 - 2021 itsallcode.org
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class MarkdownSpanConverterTest
+{
+    @ParameterizedTest(name = "Line ''{0}'' converted to HTML ''{1}''")
+    @CsvSource(
+    {
+            "'    code\n', '<pre>    code\n</pre>'",
+            "'    code', '    code'",
```

Comment:
The implementation seems to be broken here. I would expect output `<pre>    code</pre>` in this case.

@redcatbear Please create an issue if you think this is a problem.

Reply:
Actually it should be n-4 spaces in the `pre` block.

Reply:
This is more effort to fix. I created ticket https://github.com/itsallcode/openfasttrace/issues/312